config/fieldparams/minimal.go

```diff
@@ -25,6 +25,5 @@ const (
 	SlotsPerEpoch                         = 8             // SlotsPerEpoch defines the number of slots per epoch.
 	SyncCommitteeAggregationBytesLength   = 1             // SyncCommitteeAggregationBytesLength defines the sync committee aggregate bytes.
 	SyncAggregateSyncCommitteeBytesLength = 4             // SyncAggregateSyncCommitteeBytesLength defines the length of sync committee bytes in a sync aggregate.
-	MaxWithdrawalsPerPayload              = 16            // MaxWithdrawalsPerPayloadLength defines the maximum number of withdrawals that can be included in a payload.
-	WithdrawalQueueLimit                  = 1099511627776 // WithdrawalQueueLimit defines the maximum number of withdrawals queued in the state.
+	MaxWithdrawalsPerPayload              = 4             // MaxWithdrawalsPerPayloadLength defines the maximum number of withdrawals that can be included in a payload.
```

Comment:
Same here, why these changes to presents?

Reply:
The correct value for `MaxWithdrawalsPerPayload` is [4](https://github.com/ethereum/consensus-specs/blob/dev/presets/minimal/capella.yaml)
This has to be fixed or else the minimal spec test will fail